### DISCUSSION TITLE
Is that ok if i will add workshop uploading feature to my app?

### DISCUSSION THREAD

**Discussion by iBowie** [SmartlyDressedGames/Unturned-3.x-Community#2428] [Q&A] [answered]
I think this feature would be very useful for NPC Maker app, since then people could upload created NPCs via app with only one button. I'm concerned about legal stuff and all, since i have to include 'steam_api.dll', 'Steamworks.NET.dll' and 'steam_appid.txt' to my apps bundle, and also had to make an app that 'mimics' Unturned (when it opens Steam thinks that Unturned is launched). This other app closes as soon as it done uploading. Am i allowed to do this?

**Comment by SDGNelson** [comment, marked as answer]
Cool idea, yeah feel free to do so!
